config/HELICSConfig.cmake.in

```diff
@@ -13,36 +13,47 @@
 #   HELICS_BROKER -- the helics broker executable
 @PACKAGE_INIT@
 
+get_filename_component(HELICS_PREFIX_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
```

Comment:
What's the reason for doing this instead of using the PACKAGE_PREFIX_DIR generated by @PACKAGE_INIT@?

Reply:
Because that was giving me the wrong folder on Appveyor for some reason, so I am trying something different and if that works I will try to figure out why PACKAGE_PREFIX_DIR was wrong

Reply:
If it's for the find_package tests, that's probably because it's using files in the HELICS build tree for the targets, so some of the paths (bin/lib/cmake subfolders) used for an install won't exist.